src/ResourceManager/DataMigration/Commands.DataMigration/Cmdlets/NewDataMigrationSelectedDB.cs

```diff
@@ -0,0 +1,114 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="NewDataMigrationSelectedDB.cs" company="Microsoft">
+//     Copyright (c) Microsoft Corporation.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.Azure.Management.DataMigration.Models;
+
+namespace Microsoft.Azure.Commands.DataMigration.Cmdlets
+{
+    [Cmdlet(VerbsCommon.New, "AzureRmDataMigrationSelectedDB", SupportsShouldProcess = true, DefaultParameterSetName = SqlServerSqlDbParameterSet), OutputType(typeof(MigrateSqlServerSqlDbDatabaseInput))]
+    [Alias("New-AzureRmDmsSelectedDB")]
```

Comment:
If this is the old "New-AzureRmDataMigrationSqlServerSqlDbSelectedDB"? If so, you should add an alias to that old name to prevent breaking customers.

Reply:
I thought you were combining the two cmdlets - was I not understand that correctly?

Reply:
Yes, that is correct. NewDataMigrationSelectedDB.cs will work for both SqlDb and SqlMI type databases.

New-AzureRmDataMigrationSqlServerSqlDbSelectedDB is already published. Is it okay to remove it?
(With my current changes, NewDataMigrationSelectedDB.cs with SqlDb as switch parameter and New-AzureRmDataMigrationSqlServerSqlDbSelectedDB are equivalent in terms of functionality.